terraform/ec2/win/install_and_start_agent.tpl

```diff
@@ -0,0 +1,22 @@
+<powershell>
+$installDirectory = "c:\temp\cw"
```

Comment:
does this need double backslash? 
Same question on the `$cwAgentInstaller` variable.

Reply:
Not when it is in the tpl file.